tests/migration_test.go

```diff
@@ -1786,6 +1786,114 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				})
 			})
 		})
+		Context("with a host-model cpu", func() {
+			getNodeHostModel := func(node *k8sv1.Node) (hostModel string) {
+				for key, _ := range node.Labels {
+					if strings.HasPrefix(key, v1.HostModelCPULabel) {
+						hostModel = strings.TrimPrefix(key, v1.HostModelCPULabel)
+						break
+					}
+				}
+				Expect(hostModel).ToNot(BeEmpty()) // Must find node's host model
```

Comment:
You can move the comment to the description of the assert. Then we can see what's wrong without looking into the code.